projects/bnry/index.js

```diff
@@ -0,0 +1,20 @@
+const BNRY_TOKEN_CONTRACT = '0xb5090d514bcaca7dafb7e52763658844121f346d';
+const EOA_WALLET = '0xfB1dA2bA2B6c1e73e4Ace7aF2A38Fea4C289508e';
```

Comment:
this is a wallet, not a smart contract, so you have no tvl.

https://docs.llama.fi/list-your-project/what-to-include-as-tvl